compiler/testData/codegen/bytecodeText/inline/adaptedReference.kt

```diff
@@ -0,0 +1,11 @@
+// !LANGUAGE: +SuspendConversion
+// FILE: a.kt
+fun nonSuspend() {}
+
+inline suspend fun f(x: suspend () -> Unit) = x()
+
+// FILE: b.kt
+suspend fun call() = f(::nonSuspend)
+
+// @BKt.class:
+// 0 INVOKEINTERFACE kotlin/jvm/functions/Function1.invoke
```

Comment:
Minor. Maybe convert it into boxInline test? It will automatically check that there is no additional anonymous objects

Reply:
Unfortunately this check seems to not work properly for tests marked `WITH_COROUTINES` - something inside the injected `helpers` package breaks it.

Reply:
Is it possible to get rid of suspend at all, look like they are not neccessery according to https://youtrack.jetbrains.com/issue/KT-38536 issue?

Reply:
Oh right, I forgot there was a test already. Still, the check seems very unreliable - for some reason, on that test it only fails in the `JvmIrAgainstOld` configuration, even though `IrBlackBoxInline` contains a lambda class too.

Reply:
Never mind, it also triggers in `IrCompileKotlinAgainstInlineKotlin`, I just used the wrong test filter.